srcpkgs/m4ri/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'm4ri'
+pkgname=m4ri
+version=20200125
+revision=1
+build_style=gnu-configure
+configure_args="--enable-openmp"
+hostmakedepends=pkg-config
+makedepends=libpng-devel
+short_desc="Library for fast arithmetic with dense matrices over GF(2)"
+maintainer="Eloi Torrents <eloitorrentsj@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://bitbucket.org/malb/m4ri"
+distfiles="${homepage}/downloads/${pkgname}-${version}.tar.gz"
```

Comment:
It seems people prefer to not use `${homepage}`, could you change this to
`distfiles="https://bitbucket.org/malb/m4ri/downloads/${pkgname}-${version}.tar.gz"`
instead?